code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -38,9 +38,14 @@
 	///Radio connection from the air alarm
 	var/radio_filter_in
 
+	//Enables the use of plunger_act for ending the vent clog random event
+	var/clogged = FALSE
+
 	COOLDOWN_DECLARE(check_turfs_cooldown)
 
 /obj/machinery/atmospherics/components/unary/vent_scrubber/New()
+	RegisterSignal(src, COMSIG_VENT_CLOG, .proc/clog)
+	RegisterSignal(src, COMSIG_VENT_UNCLOG, .proc/unclog)
```

Comment:
This actually seems a bit like signal misuse. 

`clog()` can just be called directly from the event on the scrubber, and doesn't need a signal. 
`unclog()` can also be called from either the event when it's done, or a plunger.